src/lexer.coffee

```diff
@@ -1187,7 +1187,7 @@ exports.JS_FORBIDDEN = JS_KEYWORDS.concat(RESERVED).concat(STRICT_PROSCRIBED)
 BOM = 65279
 
 # Token matching regexes.
-IDENTIFIER = /// ^
+exports.IDENTIFIER = IDENTIFIER = /// ^
```

Comment:
Export the `IDENTIFIER` regex to allow the Prettier plugin to import it and use it eg to decide whether it needs to quote object keys or not

Reply:
We should probably export all the constants if we export any. Maybe something at the bottom like `exports.lexerConstants = { IDENTIFIER, ... }`

Reply:
Agreed this is sort of arbitrary to just export `IDENTIFIER`

I just updated the Prettier plugin to not import this but instead to just use the same utility that Prettier uses for determining whether an object key is a valid identifier name (and can thus be printed unquoted)

So correspondingly removed the export of `IDENTIFIER` here